### PR TITLE
📝 Add nextJS app example

### PR DESCRIPTION
**Pull request by maurusrv**
This project illustrates how to import the `clevertap-web-sdk` on a nextJS app.

Since the module is designed for [client-side only](https://github.com/CleverTap/clevertap-web-sdk/issues/39#issuecomment-983328291), for nextJS apps, one can't straightforwardly import the sdk like: 

```js
import clevertap from 'clevertap-web-sdk'
```

due to the library expecting for window object to be defined. See [#52](https://github.com/CleverTap/clevertap-web-sdk/issues/52)

Aside from a [workaround](https://github.com/CleverTap/clevertap-web-sdk/issues/39), one can *_asyncly import_* it when the component mounts or just before an event push:

```js
async function initializeClevertap(): Promise<CleverTap> {
  const clevertap = await import('clevertap-web-sdk')

  // clevertap.default.init(ACCOUNT_ID, TARGET_DOMAIN)
  // clevertap.privacy.push({ optOut: false })
  // clevertap.privacy.push({ useIP: false })
  clevertap.default.setLogLevel(3)

  return clevertap.default
}
```

This example app does not include the use of react context for simpler implemention or different state / context management.

### CODE REVIEW

File: example-apps/next/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Aside from a [workaround](https://github.com/CleverTap/clevertap-web-sdk/issues/39), one can *_asyncly import_* it when the component mounts or just before an event push:
+
+```js
+async function initializeClevertap(): Promise<CleverTap> {
+  const clevertap = await import('clevertap-web-sdk')
+
+  // clevertap.default.init(ACCOUNT_ID, TARGET_DOMAIN)
+  // clevertap.privacy.push({ optOut: false })
+  // clevertap.privacy.push({ useIP: false })
+  clevertap.default.setLogLevel(3)
```
Comment (narender2031) [inline example-apps/next/README.md:42]: No need to set `setLogLevel(3)`

**Review by narender2031 — CHANGES_REQUESTED** [review]

File: example-apps/next/README.md (hunk 1 above)
Comment (maurusrv) [reply example-apps/next/README.md:42]: i guess this example might imply that it is needed. I put it there to make the user see it in the devtool when they try it. Anyway, i guess I can comment that out too.

**Comment by Deveshb15** [comment]
Did this work for anyone?

**Comment by narender2031** [comment]
yes

**Comment by Deveshb15** [comment]
> yes

Did you call the function inside a useEffect?

useEffect(() => {
    initializeClevertap()
}, [])

**Comment by KambleSonam** [comment]
@maurusrv  Kindly go through the sample Nextjs web app. Let us know if you are facing any issues here.
https://clevertap-next-demo.vercel.app/
Repo Link - https://github.com/KambleSonam/clevertap-next-demo

**Comment by maurusrv** [comment]
> @maurusrv Kindly go through the sample Nextjs web app. Let us know if you are facing any issues here. https://clevertap-next-demo.vercel.app/ Repo Link - https://github.com/KambleSonam/clevertap-next-demo

@KambleSonam any differences that i miss between the demo repo and this PR? if none, it should be good as i also see the dev logs too on the demo site. 

1. it has been a year or so since i created this PR. i am still new in contributing to open source so i don't know, should i close this now? I realize maintaining setups in this repo might become a hassle and does not really make sense since frameworks like next will always update. And the info in the description above is already good for others to stumble upon.

2. also, for the same reason that it has been a year, note that the sample is not using the latest next setup. but the clevertap integration still is relevant as this is still how our setup is right now when using the sdk on our nextjs clients. one can also consider just using the clevertap api using fetch instead if they are using custom events and not the system events.

thanks!

**Pull request closed without merging**